gpMgmt/Makefile

```
@@ -6,12 +6,13 @@ SUBDIRS= sbin bin doc
 
 $(recurse)
 
-generate_greenplum_path_file:
+generate_cloudberry_path_file:
```

We can also update the rule name to address the file name change:


```suggestion
generate_cloudberry_env_file:
```